azure/pipelines/dotnet/ci.yml

```diff
@@ -136,22 +160,31 @@ steps:
       arguments: '--configuration ${{ parameters.BuildConfiguration }} --no-restore --no-build'
     condition: succeededOrFailed()
 
-- ${{ if eq(parameters.WithPack, 'true') }}:
-  - task: DotNetCoreCLI@2
-    displayName: 'DotNet Pack'
+- ${{ if eq(parameters.WithPush, 'true') }}:
+  # Push NuGets.
+  - task: NuGetCommand@2
+    displayName: 'NuGet Push'
     inputs:
-      command: 'pack'
-      packagesToPack: ${{ parameters.Solution }}
-      versioningScheme: 'off'
-      arguments: '--configuration $(BuildConfiguration) --no-restore --no-build'
+      command: 'push'
+      packagesToPush: '$(Build.ArtifactStagingDirectory)/*.nupkg;$(Build.ArtifactStagingDirectory)/*.snupkg;!$(Build.ArtifactStagingDirectory)/*.symbols.nupkg'
+      nuGetFeedType: 'external'
+      publishFeedCredentials: $(externalFeedCredentials)
+
+  # The above task does publish the .snupkgs, but Visual Studio can't seem to consume them from the artifact feed yet?
+  # Therefore, also publish the .pdb to the SymbolServer of this organization.
+  # Users can get rich debugging experience by adding sln-sup as VSTS symbol server in addition to dlw-pltn.
```

Comment:
You have to add your own project as VSTS symbol server, not just 'sln-sup' or dlw-pltn?
Also: I think it should be sln-sup**p**?